blobstore/src/main/java/org/jclouds/blobstore/options/ListContainerOptions.java

```diff
@@ -44,18 +44,20 @@
    private String prefix;
    private boolean recursive;
    private boolean detailed;
+   private int listType = 1;
 
    public ListContainerOptions() {
    }
 
    ListContainerOptions(Integer maxKeys, String marker, String dir, boolean recursive,
-            boolean detailed, String prefix, String delimiter) {
+            boolean detailed, String prefix, String delimiter, int listType) {
```

Comment:
Sorry I missed this earlier, but why does the portable abstraction need to know the S3 list type?  This looks like a layering violation.